atomics/T1562.001/T1562.001.yaml

```diff
@@ -574,17 +574,6 @@ atomic_tests:
       cmd /c #{DefenderControlExe} /E | Out-Null
     name: powershell
     elevation_required: true
-- name: Disable Windows Defender Tamper Protection
```

Comment:
Ref: to the issue #1846 
From testing, changing the reg key value under 'Policies\Microsoft\Windows Defender\Feature' no longer disables TamperProtection as long as 'RealTimeMonitoring' is enabled.

'RealTimeMonitoring' reg key is owned by system. Tried using PsExec tool to run 'regedit' as 'NT authority \ system' user to change ownership and value of reg key for 'RealTimeMonitoring' and 'TamperProtection', but that also wont work on a system on which  'RealTimeMonitoring' is already enabled.